noetic/distribution.yaml

```diff
@@ -11253,6 +11253,26 @@ repositories:
       url: https://github.com/ros-industrial/ur_msgs.git
       version: melodic-devel
     status: maintained
+  ur_robot_driver:
+    doc:
+      type: git
+      url: https://github.com/UniversalRobots/Universal_Robots_ROS_Driver.git
+      version: master
+    release:
+      packages:
+      - controller_stopper
```

Comment:
Yes, the packages was indeed meant to be a generic node to interact with the `controller_manager`. Since I had implemented something similar using a KUKA robot in the past, I thought it would be good to make this a generic node, hence the generic name. However, since I'm not too deeply involved into `ros_control` development I think it would be better to keep it here at the moment. I assume the location of this package could be changed again, later.

Reply:
I do agree that the name may be too generic. If we're going to keep it in this repo, maybe we could change the name to be less generic. If it does move over to `ros_control`, then maybe we can go back to the generic name.

Reply:
Thanks for the feedback. Although I do not fully agree, I also understand the concerns raised.

For the record: I've created an issue at https://github.com/ros-controls/ros_control/issues/510 but we'll be thinking about a more suitable name in parallel.

Reply:
I'm really looking forward to this release :+1: 
would it be an option to `noetic.ignore` the `controller_stopper` in this first release and continue with the other packages - and then resolve the issue around `controller_stopper` in the next iteration?

Reply:
Leaving out the controller_stopper is not really an option, as it is a runtime dependency for the driver (at least from its launchfiles).

We've been discussing this internally and we basically see two ways forward: Either we merge the controller_stopper into the driver's package, as we did for ROS2 (as it is a bit more UR-specific there) or we leave it with the current name and publish it like that. The latter would leave the option to migrate the package to ros_control at some point. @audrow do you think the second option would be possible? Otherwise I will go ahead and drop the standalone package.